javacutil/src/main/java/org/checkerframework/javacutil/CollectionUtils.java

```diff
@@ -23,4 +28,119 @@ protected boolean removeEldestEntry(Map.Entry<K, V> entry) {
       }
     };
   }
+
+  ///
+  /// Temporary utility methods
+  ///
+
+  // TODO: After plume-util 1.6.6 is released, use these methods from it.
```

Comment:
@mernst this comment seems to contradict https://github.com/typetools/checker-framework/pull/5791#issuecomment-1520675093